packages/services/package.json

```diff
@@ -21,7 +21,7 @@
     "commander": "^4.0.0",
     "ganache-core": "2.4.0",
     "http-proxy": "1.18.0",
-    "ipfs": "^0.40.0",
+    "ipfs": "^0.43.3",
```

Comment:
Not sure we should check this in yet since it breaks the DApp unit tests :(

Reply:
reverted